frontend/src/DmarcReportTable.js

```diff
@@ -0,0 +1,316 @@
+import React, { useRef, useState } from 'react'
+import styled from '@emotion/styled'
+import { useTable, usePagination, useSortBy } from 'react-table'
+import { array, string } from 'prop-types'
+import {
+  Box,
+  Button,
+  Collapse,
+  Icon,
+  IconButton,
+  Input,
+  Select,
+  Stack,
+  Text,
+} from '@chakra-ui/core'
+import { Trans } from '@lingui/macro'
+import { t } from '@lingui/macro'
+import { useLingui } from '@lingui/react'
+
+import WithPseudoBox from './withPseudoBox'
+
+const Table = styled.table`
+width: calc(100% - 2px);
+border-collapse: collapse;
+border: 1px solid #ccc;
+
+caption {
+  width: 100%;
+}
+
+th {
+  color: black;
+  font-weight: bold;
+}
+
+td, th {
+  padding: 6px;
+  border: 1px solid #ccc;
+  text-align: left;
+}
+
+.pagination {
+  padding: 0.5rem;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
+  clip: rect(1px, 1px, 1px, 1px);
+  white-space: nowrap; /* added line */
+}
+
+// Remove outside borders from cells, let table handle border
+@media screen and (min-width: 761px) {
+  tr:first-type-of td {
+    border-top: 0;
+  }
+  tr td:first-type-of, th:first-type-of {
+    border-left: 0;
+  }
+  tr:last-child td {
+    border-bottom: 0;
+  }
+  tr td:last-child, th:last-child {
+    border-right: 0;
+  }
+}
+
+@media screen and (max-width: 760px) {
+  table,
+  thead,
+  tbody,
+  th,
+  td,
+  tr {
+    display: block;
+  }
+  tr { border: 1px solid #ccc; }
+  td {
+    border: none;
+    border-bottom: 1px solid #ccc;
+    position: relative;
+    padding-left: 50%;
+  }
+  td: before {
+    position: absolute;
+    top: 6px;
+    left: 6px;
+    width: 45%;
+    padding-right: 10px;
+    white-space: nowrap;
+  }
+  ${(props) =>
+    props.flatHeaders.slice(1).map((headerObj, index) => {
+      return `
+        td:nth-of-type(${index + 1}):before {
+          content: '${headerObj.Header}';
+          font-weight: bold;
+        }
+      `
+    })}
+`
```

Comment:
This component isn't getting it's styles from our [theme](https://github.com/canada-ca/tracker/blob/d2688b6f7841648c88cd3f40e31c64d6cd0ddbd0/frontend/src/index.js#L10). Chakra UI is [built around the idea of a theme](https://chakra-ui.com/getting-started), [an idea that it inheirits from Styled-System](https://styled-system.com/getting-started#theming) by way of building on top of it. That theme supplies all the breakpoints widths, colours and other stuff, which this component is defining for itself, creating a little "style island". In service of "gathering feedback" this is fine, but needs a follow up PR to switch it over to drawing it's values from the theme or this is going to get out of hand quickly.